analysis/analysis.py

```diff
@@ -0,0 +1,136 @@
+#!/usr/bin/python
+
+from matplotlib.pyplot import figure, show
+import datetime
+import dateutil.parser
+import json
+import numpy as npy
+import os
+import pprint
+import sys
+import urllib
+#import pylab as plt
+
+if (len(sys.argv) == 1 or sys.argv[1] == "-h"):
+    print "Usage: " + sys.argv[0] + " json_output [csv_output]"
+    print
+    print " `-> Print the analysis as json into the json_output file, and,"
+    print "     if another file is given, write as CSV the informations concerning"
+    print "     the maximum amount of WiFi devices measured by day, by captor, into"
+    print "     the csv_output file."
+    print
+    exit()
+
+secret_json = os.path.dirname(os.path.abspath(__file__)) + "/../PRIVATE.json"
+with open(secret_json) as configuration_json:
+    configuration = json.load(configuration_json)
+
+pp = pprint.PrettyPrinter(indent=4)
+allinfos_url = configuration["data_source"] + "/allPlacesInfos"
+allinfos = json.loads(urllib.urlopen(allinfos_url).read())
+allcaptors = []
+base = datetime.datetime.today()
+
+opening_hours_url = configuration["data_source"] + "/sensor/getAll?s=" + configuration["secret"]
+opening_hours = json.loads(urllib.urlopen(opening_hours_url).read())
```

Comment:
For clarity reasons, you should call this variable `sensors`, because it actually holds all sensors' details (even if you only use their `start_hour`/`stop_hour`).